plugins/nextjs/src/pageGenerator/pageGenerator.test.ts

```diff
@@ -8,6 +8,11 @@ describe('pageGenerator', () => {
 
     await plugin.run({
       generator: 'page',
+      itemName: 'movie-magic',
+      rootDir: '../../',
```

Comment:
This is a bit of a cop-out. Technically rootDir should be `'test-output'`. I think this just arranges the test to read workspaces from the real root directory instead of a blank `test-output` directory. A more robust solution would be to write generators so that they can generate in blank directories.

I will rework this part. Hang tight!